modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase

```diff
@@ -2666,4 +3096,113 @@ definition {
 			value1 = "7.00");
 	}
 
+	@description = "This is a test for COMMERCE-9459. ViewShipmentCarrierTrackingURL"
+	@priority = "3"
+	test ViewShipmentCarrierTrackingURL {
```

Comment:
Rename it to CanViewShipmentTrackingURL

Reply:
Ok

Reply:
This is not resolved yet, please rename it